mobile/lib/modules/login/services/oauth.service.dart

```diff
@@ -33,7 +34,8 @@ class OAuthService {
           url: result,
         ),
       );
-    } catch (e) {
+    } catch (e, stack) {
+      log.severe("Error performing oAuthLogin: ${e.toString()}", e, stack);
```

Comment:
Man, how did we not have a log here before. Just suppressing it? This hopefully helps capture the problems!